vivo-project/Vitro
# [VIVO-1674] Improve quality of thumbnails created -- Maint rel 1.9

**[JIRA Issue](https://jira.duraspace.org/browse/VIVO-1674)**: VIVO-1674

This pull request is for maint-rel-1.9

# What does this pull request do?
Address issue with quality of thumbnails created when uploading an image

# What's new?
Use Twelvemonkeys plugin for scaling thumbnails
Adds additional logic to the jpg creation to allow us to set jpg compression level
Adds in changes and dependencies from 1.10 into 1.9.3

# How should this be tested?
You can reproduce the issue by upload an image to VIVO. Observe the thumbnail that is created... it likely looks like it was cut out of a Norm's Diner menu in the 80s and somehow mimeographed onto the internet, i.e. not good.
![image](https://user-images.githubusercontent.com/1811461/52675049-b83cf500-2ee2-11e9-807c-476b3fc49f01.png)

Build Vitro/VIVO with the changes. Thumbnails should now be much better looking.
![image](https://user-images.githubusercontent.com/1811461/52675031-a3606180-2ee2-11e9-9d31-aa0eb70e4661.png)


# Additional Notes:
The twelvemonkeys plugin for ImageIO is already being included in [Vitro dependencies](https://github.com/vivo-project/Vitro/blob/f2656ba3d8ae67359567d5aecd92eb67ba743c62/dependencies/pom.xml#L74)

As always, improvements are welcome.

# Interested parties
@gneissone @awoods @grahamtriggs
